package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbeat",
-  "version": "1.0.0",
```

Comment:
I would leave that to @jonathan-gramain to see if we wants to bump the version but I doubt it.